src/frontend/pages/SendLetter/SendLetter.tsx

```diff
@@ -104,6 +105,10 @@ export const SendLetter: React.FunctionComponent<RouteComponentProps> = () => {
               <label htmlFor="content">Message</label>
               <textarea required id="content" placeholder="Your message" rows={10} />
             </p>
+            <p className="field">
+              <label htmlFor="email">Email (optional)</label>
```

Comment:
Although this is unnecessary, it's good that you update this file to keep it consistent with the one on the public site.